src/library/enqueue.cpp

```diff
@@ -90,13 +90,18 @@ FFTStockhamAction::FFTStockhamAction(clfftPlanHandle plHandle, FFTPlan * plan, c
 }
 
 
-
 FFTAction::FFTAction(FFTPlan * fftPlan, clfftStatus & err)
     : plan(fftPlan)
 {
     err = CLFFT_SUCCESS;
 }
 
+
+FFTAction::~FFTAction()
```

Comment:
what is the need for empty destructor?

Reply:
It's a base class, with virtual functions, so it needs the declaration so the destructor of the derived classes gets called. The implementation here is to make the linker happy.